index.html

```diff
@@ -14,7 +14,7 @@
 
         <div class="page-heading">How Istanbul works</div>
         <p>
-          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
+          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
```

Comment:
Nope. ES2015 is literally so last year

On 3 Aug 2016 20:30, "Kent C. Dodds" <notifications@github.com> wrote:

> In index.html
> <https://github.com/istanbuljs/istanbuljs.github.io/pull/10#discussion_r73404303>
> :
>
> > @@ -14,7 +14,7 @@
> >
> >          <div class="page-heading">How Istanbul works</div>
> >          <p>
> > -          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
> > +          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
>
> I'm guessing you're joking, but I can't tell.... This text should be
> changed to ES2015+
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/istanbuljs/istanbuljs.github.io/pull/10/files/feaed766c653c92d8a451b8ef5e8e8c49b371024#r73404303>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZQTMfzh4aRjnOsYKRcrQJOyRotV4ISks5qcOxFgaJpZM4Jb-yd>
> .
>